README.md

```diff
@@ -122,6 +123,8 @@ Will send a log to Logz.io that looks like this:
 ```
 
 ### Release notes
+ - 1.0.8
+   - added `compressRequests` parameter to enable gzip compression of the logs before they are sent.
```

Comment:
added option to inject system property value into additionalFields, logzioUrl and token.

Please add "system property" in to the "token" parameter description.